osm2streets/src/transform/intersection_geometry.rs

```diff
@@ -49,21 +54,29 @@ pub fn generate(streets: &mut StreetNetwork, timer: &mut Timer) {
                         road.center_line.last_pt()
                     };
                     set_polygons.push((i.id, Circle::new(pt, Distance::meters(3.0)).to_polygon()));
-
-                    // Also don't attempt to make Movements later!
```

Comment:
This is some weird old A/B Street assumption we can get rid of now. When intersection geometry broke at a traffic signal, my old code for figuring out signal timing broke. I'll workaround that elsewhere.